src/ZendCodingStandard/Sniffs/Commenting/VariableCommentSniff.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Copied from Squiz_Sniffs_Commenting_VariableCommentSniff
+ * with modifications:
+ *  - use suggested types from ZendCodingStandard\CodingStandard
+ *  - skip normal comments if there is one blank line after the comment
+ */
+
+namespace ZendCodingStandard\Sniffs\Commenting;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\AbstractVariableSniff;
+use ZendCodingStandard\CodingStandard;
+
+class VariableCommentSniff extends AbstractVariableSniff
+{
+    /**
+     * @inheritDoc
```

Comment:
But ... with our new standards, we are going to remove these PHPDocs, because these are not giving us any additional information.